ios/Runner/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIStatusBarHidden</key>
```

Comment:
Em princípio se apagares isto, a linha de código que mete a status bar novamente visível deixará de ser precisa (caso se pretenda que esteja visível, que é o comportamento por omissão).

Reply:
I'll test it out, thank you.